docs/content/ActionMenu.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: ActionMenu
+---
+
+An `ActionMenu` is a simple ActionList based component for creating a menu of actions that expands through a trigger button.
```

Comment:
I try to avoid using words like `simple` and `just` because it can make readers feel discouraged if they don't find it simple ([link](https://jameshfisher.com/2017/02/22/dont-use-simply/)).
```suggestion
An `ActionMenu` is an ActionList-based component for creating a menu of actions that expands through a trigger button.
```